src/zc/buildout/easy_install.py

```diff
@@ -882,6 +883,45 @@ def _copyeggs(src, dest, suffix, undo):
 
     return result[0]
 
+
+_develop_distutils_scripts = {}
+
+
+def _detect_distutils_scripts(directory):
+    """Record detected distutils scripts from develop eggs
+
+    ``setup.py develop`` doesn't generate metadata on distutils scripts, in
+    contrast to ``setup.py install``. So we have to store the information for
+    later.
+
+    """
+    dir_contents = os.listdir(directory)
+    egginfo_filenames = [filename for filename in dir_contents
+                         if filename.endswith('.egg-link')]
+    if not egginfo_filenames:
+        return
+    egg_name = egginfo_filenames[0].replace('.egg-link', '')
+    marker = 'EASY-INSTALL-DEV-SCRIPT'
+    scripts_found = []
+    for filename in dir_contents:
+        dev_script_content = open(os.path.join(directory, filename)).read()
```

Comment:
Ok, I'll add a "with" here. @mgedmin: I'll also add an `os.path.isfile()` check. It shouldn't be necessary, but I *did* have to add such a check elsewhere already because of some python 3 `__pycache__` or so directory.